docs/interface.md

```diff
@@ -0,0 +1,75 @@
+# The Interface

```

Comment:
t looks like much of this information is a duplicate of https://github.com/exercism/automated-tests/blob/master/docs/interface.md. I'd prefer not to duplicate the information and instead link to the aforementioned document, which goal is to be a shared reference.

Reply:
It is in fact a complete duplicate. I'll remove them and go with a link :)

Reply:
Brilliant!